### PR TITLE
Adding PySimpleGUIWx

### PR DESCRIPTION
**Pull request by Silun**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by Silun** [comment]
@conda-forge-admin, please rerender

**Comment by Silun** [comment]
I think this is ready for review, if anybody would take a look please.

File: recipes/PySimpleGUIWx/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{% set name = "pysimpleguiwx" %}
+{% set version = "0.15.0" %}
+{% set styledName = "PySimpleGUIWx" %}
```
Comment (npavlovikj) [inline recipes/PySimpleGUIWx/meta.yaml]: FYI, you don't need to use `styledName` - you can have `name = "PySimpleGUIWx` and use `name|lower` when needed.

File: recipes/PySimpleGUIWx/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - wxpython
```
Comment (npavlovikj) [inline recipes/PySimpleGUIWx/meta.yaml]: Are you sure `wxpython` should be in the `host` section and not `run`? Usually only `pip` and `python` belong in the `host` section, and all other dependencies in `run`.

File: recipes/PySimpleGUIWx/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
```
Comment (npavlovikj) [inline recipes/PySimpleGUIWx/meta.yaml]: Remove empty line.

File: recipes/PySimpleGUIWx/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - PySimpleGUIWx
+  requires:
```
Comment (npavlovikj) [inline recipes/PySimpleGUIWx/meta.yaml]: When you add `wxpython` to the `run` section, no need to use `requires` here.

**Comment by Silun** [comment]
Thank you for the detailed comments, you certainly were right. I applied your suggestions.

**Comment by npavlovikj** [comment]
Thank you @Silun !

**Pull request merged**